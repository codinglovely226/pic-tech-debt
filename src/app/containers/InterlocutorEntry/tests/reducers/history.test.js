/* eslint-disable */

import historyReducer from '../../reducers/history';

describe('historyReducer', () => {
  var mockedDate = new Date(2018, 3, 29, 6, 22, 20, 817);
  global.Date = jest.fn(() => mockedDate);
  it('returns the initial state', () => {
    expect(historyReducer(undefined, {})).toMatchSnapshot();
  });
});
