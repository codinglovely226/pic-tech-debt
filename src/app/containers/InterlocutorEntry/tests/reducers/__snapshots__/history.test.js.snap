exports[`historyReducer returns the initial state 1`] = `
Object {
  "agents": Object {},
  "features": Object {
    "descriptors": "",
    "outcome": "",
    "pastDescriptors": Object {},
    "when": "",
  },
  "investigations": Object {},
  "patient": Object {
    "age": null,
    "ageGroup": null,
    "ageUnits": null,
    "ageYears": null,
    "animalType": null,
    "category": null,
    "circumstance": null,
    "gender": null,
    "localityAddress1": null,
    "localityCity": null,
    "localityCountry": null,
    "localityGeometry": null,
    "localityISOCountry": null,
    "localityLocation": null,
    "localityName": null,
    "localityPlaceId": null,
    "localityPostcode": null,
    "localityState": null,
    "localityTelephone": null,
    "medicalRecordNumber": null,
    "medicalRecordNumberEnc": null,
    "name": null,
    "nameEnc": null,
    "occupation": null,
    "outcome": null,
    "poisonSeverityScore": null,
    "pregnantLactating": null,
    "pregnantStatus": null,
    "pregnantTrimester": null,
    "pregnantTrimesterUnknown": null,
    "riskAssessment": null,
    "topic": null,
    "weight": null,
    "weightUnits": null,
  },
  "treatments": Object {
    "pastTreatments": Object {},
    "referrals": Array [],
    "treatments": Array [],
  },
}
`;
